backend/src/main/resources/application-dev.yaml

```diff
@@ -22,6 +22,3 @@ simple-report:
   data-hub:
     upload-enabled: ${DATAHUB_UPLOAD_ENABLE:false}
     upload-url: "https://prime-data-hub-test.azurefd.net/api/reports?option=SkipSend"  # ValidatePayload, SkipSend, SkipInvalidItems
-okta:
-  oauth2:
-    client-id: 0oa61vdmqd56921Ka4h6
```

Comment:
this same property exists with the same definition in `application-okta-dev.yaml` (i don't know why, if that was my doing i sincerely apologize.) but i think it _does_ belong in `-okta-dev`. so i'm not convinced removing this will do anything, but i'm not opposed to it either.

Reply:
We don't use the okta-dev profile for local development, though--it's just for dev.simplereport.gov (and test and QA at one point, but not now).

(Also, experimentally this did produce the error when using the `dev` profile locally.)

Reply:
Ah, right.